src/enforce/chainables/compound/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import runners from '../../runners';
+import { Errors } from 'Constants';
+import { runtimeError } from 'Helpers';
+
+/**
+ * Run group of tests using test runner. (e.g. `anyOf`)
+ *
+ * @private
+ * @param {string} group - name of test runner
+ * @param {object} tests
+ * @return {object} enforce object
+ */
+function compound(value: AnyValue, group: string, tests: Tests, allRules: Rules = {}): EnforceSelf {
+    if (this.valid === false) {
+        return this;
+    }
+
+    this.valid = runners[group](value, tests, allRules);
+
+    if (this.valid !== true) {
```

Comment:
Why not:
```js
if (!this.valid)
```

Same in `chainables/single`.

Reply:
It needs to explicitly pass.